release.sh

```diff
@@ -32,13 +38,14 @@ if git log --skip 1 origin/master..origin/develop|grep '^commit '; then
   python dev_tools.py bump_version dev
   git add flexget/_version.py
   git commit -m "Prepare v`python dev_tools.py version`"
+  git branch -f develop HEAD
 
   # master branch should be at the release we tagged
   git branch -f master ${VERSION}
   # If either of the new branches are not fast forwards, the push will be rejected
   git push origin master develop
   # Make sure our branches push before pushing tag
-  git push --tags
+  #git push --tags
```

Comment:
Why not push tags?

Reply:
Oops, just forgot to re-enable after testing. Good catch.